.binder/postBuild

```diff
@@ -0,0 +1,33 @@
+MONGOTOOLS=mongodb-database-tools-ubuntu1804-x86_64-100.7.4
+MONGOTOOLS_FILE=$MONGOTOOLS.tgz
+MONGOTOOLS_LINK=https://fastdl.mongodb.org/tools/db/$MONGOTOOLS_FILE
+
+MONGODB=mongodb-linux-x86_64-ubuntu1804-6.0.8
+MONGODB_FILE=$MONGODB.tgz
+MONGO_LINK=https://fastdl.mongodb.org/linux/$MONGODB_FILE
+
+# Download the MongoDB and Mongorestore
+wget $MONGO_LINK 
+tar -zxvf $MONGODB_FILE
+
+wget $MONGOTOOLS_LINK
+tar -zxvf $MONGOTOOLS_FILE
+
```

Comment:
```suggestion
rm -rf $MONGOTOOLS_FILE

```
It's a nitpick - I think it looks 'cleaner' to users if this is not visible when they start their Binder session!
```